index.js

```diff
@@ -59,6 +61,10 @@ module.exports = function wdm(compiler, opts) {
     context.state = true;
   }
 
+  if (options.writeToDisk) {
```

Comment:
Can we use array of regex/glob, in real world use case not all files should be written on dist, only assets manifest and other static generation files, i.e. i have robots.txt and human.txt which generated using webpack plugin. Also we have SRI plugin which put file on disk with integrity and reading by server for include scripts/styles in page. This would solve 99% of problems.

Reply:
@shellscape sounds great! callback is good universal solution :+1:

Reply:
@evilebottnawi done, see latest changes